lib/rubocop/cop/rspec/repeated_example_group_body.rb

```diff
@@ -75,6 +75,10 @@ def add_repeated_lines(groups)
           groups.map { |group| [group, repeated_lines - [group.first_line]] }
         end
 
+        def signature_keys(group)
+          [metadata(group), body(group)]
```

Comment:
the `Example` node already has `metadata` and `implementation` methods, so you might be able to reuse them

Reply:
I thought about it firstly, but it is `ExampleGroup` in the current context, not `Example` and I gave the idea to reuse `Example` methods, it will confuse domain concepts. Another option is to implement methods in `ExampleGroup` wrapper and use them. What do you think @Darhazer?

Reply:
While it is great to extend the Language and the Node concepts in rubocop-rspec with such things, let keep it outside of the current scope. Feel free to open a PR with related refactorings later if you'd like